ocm/__init__.py

```diff
@@ -225,6 +227,16 @@ class Label:
     signing: bool = False
     merge: LabelMerge | None = None
 
+    def normalise(self) -> list[dict]:
```

Comment:
codestyle/nit: how about making this a property, and call it `normalised`? Rationale: a method `normalise` might be expected to actually change the underlying object (which it does not).

Edit: I still would vote for calling it `[as]_normalised`, but making it a property will make it inconsistent w/ other, similar, methods where we do need to pass-in parameters